packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -72,6 +74,7 @@ import {
 import { Schema as BrowserBuilderSchema } from './schema';
 
 const cacheDownlevelPath = cachingDisabled ? undefined : findCachePath('angular-build-dl');
+const {bold, red, yellow} = terminal;
```

Comment:
`terminal` is deprecated.  For now, probably drop the color.  Once structured diagnostic messaging is introduced, this can be handled on the consumer side (the CLI itself).

Reply:
Removed. This is a little weird since the older plugin (when not using differential builds) still uses those colors yet now the new differential build budgets won't.

Reply:
You can add the warnings/errors into the corresponding arrays within the webpack stats.  That would guarantee they would be displayed identically.

Reply:
Good point. Updated to use `compilation.warnings` and `compilation.errors`.